clr-installer/main.go

```diff
@@ -221,7 +222,13 @@ func execute(options args.Args) error {
 		if cf, err = conf.LookupDefaultConfig(); err != nil {
 			return err
 		}
+	} else if network.IsValidURI(options.ConfigFile, options.AllowInsecureHTTP) {
+		if cf, err = network.FetchRemoteConfigFile(options.ConfigFile); err != nil {
+			return err
```

Comment:
We need to output a message to the user here otherwise the just get an exit status number.
```suggestion
			fmt.Printf("Cannot acesss configuration file %q : %s\n", options.ConfigFile, err)
			return err
```